loom.config.ts

```diff
@@ -26,8 +26,10 @@ import type {} from '@shopify/loom-plugin-jest';
 // eslint-disable-next-line import/no-default-export
 export default createPackage((pkg) => {
   pkg.entry({root: './src/index.ts'});
+  pkg.entry({name: 'token-list', root: './src/token-list.ts'});
   pkg.use(
     buildLibrary({
+      rootEntrypoints: false,
```

Comment:
As discussed, we should spin `rootEntrypoints: false` and removing the references to root level `index.* files in the various ignore files into a separate PR that targets main.

Reply:
Sound good 👍  I created this issue for tracking https://github.com/Shopify/polaris-react/issues/4807